src/heaplayers/heaps/top/mallocheap.h

```diff
@@ -54,7 +54,7 @@ namespace HL
 
     inline void *malloc (size_t sz)
     {
-#ifdef SERVER_MODE
+#if defined(SERVER_MODE) && !defined(_MSC_VER)
```

Comment:
what does `_MSC_VER` mean here?

Reply:
It means `WINDOWS platform (Visual Studio circumstance)'. This file doesn't have macro `WINDOWS` so we cannot add `#if !defined(WINDOWS)`.

Reply:
Ah, Thanks.